src/test/java/edu/ucsb/cs156/courses/controllers/PSCourseControllerTests.java

```diff
@@ -1040,6 +1380,35 @@ public void api_courses__user_logged_in__delete_course_that_does_not_exist() thr
     assertEquals("PersonalSchedule with id 1 not found", json.get("message"));
   }
 
+  // start of new test 9
```

Comment:
does this go on? I feel like I may have missed one.

So I'm going to stop and trust you to find the rest if there are any.

Reply:
Sorry about that! I have now removed all of them.